<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSongWindow</class>
 <widget class="QMainWindow" name="EditSongWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit song</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save_item_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_song_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bridge_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add bridge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="song_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter song title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_couplet_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add couplet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_item_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_chour_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add chour</string>
    </property>
   </widget>
   <widget class="QListWidget" name="song_list">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
